development-guide.md

```diff
@@ -125,6 +125,10 @@ This operation should report all columns in the destination table, including Fiv
 - This operation might be requested for a table that does not exist in the destination. In that case, it should NOT fail, simply ignore the request and return `success = true`.
 - `utc_delete_before` has millisecond precision.
 
+#### AlterTable
+- This operation should exercise change of primary key columns, addition of columns, change of data types, etc. 
```

Comment:
@fivetran-rishabhghosh Ending with etc. is very vague. Can you provide more specific operations?